style.css

```diff
@@ -85,6 +85,251 @@
+    align-items: flex-start;
+    padding: 1px;
+    gap: 12px;
+    width: 295px;
+    height: 80px;
+  }
+
+  .card-header-text {
+    width: 295px;
+    height: 36px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+    color: #172b4d;
+    margin: 0;
+  }
+
+  .card-header-frame {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 1px;
+    gap: 12px;
+    width: 236px;
+    height: 32px;
+  }
+
+  .card-header-frame-client {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 8px 1px;
+    width: 55px;
+    height: 32px;
+  }
+
+  .card-header-frame-client-text {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    color: #344563;
+  }
+
+  .card-header-frame-role {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 8px 1px;
+    width: 88px;
+    height: 32px;
+  }
+
+  .card-header-frame-role-text {
+    width: 88px;
+    height: 16px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    color: #7a869a;
+  }
+
+  .card-header-frame-year {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 8px 1px;
+    width: 29px;
+    height: 32px;
+  }
+
+  .card-header-frame-year-text {
+    width: 29px;
+    height: 16px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    color: #7a869a;
+  }
+
+  .card-header-frame-counter {
+    width: 8px;
+    height: 8px;
+    background: #c1c7d0;
+  }
+
+  .card-description {
+    width: 295px;
+    height: 76px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .card-tags {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 1px;
+    gap: 8px;
+    width: 203px;
+    height: 24px;
+  }
+
+  .card-tag {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 4px 12px;
+    height: 24px;
+    background: #ebebff;
+    border-radius: 8px;
+  }
+
+  .card-tag-text {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #6070ff;
+  }
+
+  .card-button {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px;
+    gap: 10px;
+    width: 124px;
+    height: 48px;
+  }
+
+  .action {
+    width: 295px;
+    height: 72px;
+  }
+
+  .about-me {
+    background: #fff;
+    width: 375px;
+    height: 1125.15px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 24px;
+  }
```

Comment:
- [ ] Nice implementation of changes required by the previous reviewer, please make sure your margin, padding and overall spacing are the same as the Figma template.
 
# your design👇
![last](https://user-images.githubusercontent.com/66992901/191830224-a95ddd93-db23-41a8-b2a6-d446d5485cea.PNG)

# Figma design 👇
![figma2](https://user-images.githubusercontent.com/66992901/191830279-3ddedafe-686f-4ea9-b680-4107a91b28e3.PNG)